ladybug_geometry/geometry3d/face.py

```diff
@@ -1201,22 +1205,35 @@ def extract_rectangle(self, tolerance):
         """Extract top and bottom line segments of a rectangle within this Face.
 
         This method will only return geometry if:
-            1. There are no holes in the face.
-            2. The face is not parallel to the World XY plane.
-            3. There are two parallel edges to this face, which are either
-                oriented horizontally or vertically.
-            4. There must be enough overlap between these edges for a rectangle
-                to be drawn between them.
+
+        1) There are no holes in the face.
+
+        2) The face is not parallel to the World XY plane.
+
+        3)
+            There are two parallel edges to this face, which are either
```

Comment:
The extra line break here is pretty awkward and I don't think it's worth making the code for the list here inconsistent just to make the docs look better. Does changing it to a bulleted list have any effect?  If all else fails, I would prefer a single sentence with commas over a numbered list.

Reply:
Fixed!